dodola/services.py

```diff
@@ -151,6 +153,14 @@ def regrid(x, out, method, storage, weights_path=None, target_resolution=1.0):
         target_resolution=target_resolution,
         weights_path=weights_path,
     )
+
+    if method != "conservative":
+        lon_name = ds.cf.standard_names["longitude"][0]
+        lat_name = ds.cf.standard_names["latitude"][0]
+        ds = ds.rename({"x": lon_name, "y": lat_name})
+        ds[lat_name] = np.unique(ds[lat_name].values)
+        ds[lon_name] = np.unique(ds[lon_name].values)
```

Comment:
I if we're invoking numpy, I feel like all of this logic added to `services.py` instead needs to be implemented in `dodola/core`. My reasoning is that this renaming is very closely related to regrid implementation details (and our use of `xesmf`. The isn't responsible for this level of detail. I'd argue we should put this closer to where we use `xesmf` in `dodola/core`, because that's the module responsible for this implementation.